### PR TITLE
plugins: auto-recovery

### PR DESCRIPTION
**Pull request by jmank88**
https://smartcontract-it.atlassian.net/browse/BCF-2112

Requires:
- https://github.com/smartcontractkit/chainlink-relay/pull/108

Note: This PR disables the lower-priority median plugin for now.

### CODE REVIEW

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ env:
   ENV_JOB_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink-tests:${{ github.sha }}
   CHAINLINK_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink
   TEST_SUITE: smoke
-  TEST_ARGS: -test.timeout 30m
+  TEST_ARGS: -test.timeout 10m
```
Comment (jmank88) [inline .github/workflows/integration-tests.yml:18]: This usually runs in 8m. Not sure if 10m is too close, but haven't seen it time out yet.

File: core/services/relay/relay.go [hunk 2]
```diff
@@ -40,64 +40,51 @@ type RelayerExt interface {
 	SendTx(ctx context.Context, chainID, from, to string, amount *big.Int, balanceCheck bool) error
 }
 
-var _ loop.Relayer = (*RelayerAdapter)(nil)
+var _ loop.Relayer = (*relayerAdapter)(nil)
 
-// RelayerAdapter adapts a [types.Relayer] and [RelayerExt] to imlement [loop.Relayer].
-type RelayerAdapter struct {
+// relayerAdapter adapts a [types.Relayer] and [RelayerExt] to imlement [loop.Relayer].
```
Comment (krehermann) [inline core/services/relay/relay.go:45]: typo `imlement`

File: plugins/chainlink.Dockerfile [hunk 3]
```diff
@@ -36,7 +36,7 @@ COPY --from=buildgo /go/bin/chainlink /usr/local/bin/
 COPY --from=buildgo /go/bin/chainlink-solana /usr/local/bin/
 ENV CL_SOLANA_CMD chainlink-solana
 COPY --from=buildgo /go/bin/chainlink-median /usr/local/bin/
-ENV CL_MEDIAN_CMD chainlink-median
+#ENV CL_MEDIAN_CMD chainlink-median #TODO BCF-2112 re-enable median plugin
```
Comment (krehermann) [inline plugins/chainlink.Dockerfile:39]: is BCF-2112 the right ticket number? i thought that was for recovery

Comment (jmank88) [reply plugins/chainlink.Dockerfile:39]: I left it as part of that open ticket, hoping to fix it in a quick follow-up. Think it should be a separate?

Comment (krehermann) [reply plugins/chainlink.Dockerfile:39]: no, it's fine, just checking

**Review by krehermann — APPROVED** [review]

**Pull request merged**